docs/hacking/install.rst

```diff
@@ -186,64 +171,139 @@ system that Docker can be installed on:
 
    .. code-block:: bash
 
-      docker logs nsqd
+      sudo docker logs nsqd
 
    For more on how to use Docker see the `Docker website`_.
 
 
 .. _Docker website: https://www.docker.com/
 
 
-Running h
----------
+Installing the gulp command
```

Comment:
I'm not sure I understand why this is here. It has no context and more importantly it's not necessary. You don't need gulp installed globally to run `make dev`.